rep-0003.rst

```diff
@@ -281,6 +281,68 @@ Build System Support:
 
 - Same as Indigo
 
+Melodic Morenia (May 2018 - May 2023)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Artful (17.10)
+- Ubuntu B (18.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 28
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Targeted Languages:
+
+- C++11
+- Python 2.7
+
+  - Python 3.6 not required, but testing against it is recommended
```

Comment:
Some targeted platforms ship Python 3.5 (e.g. [debian stretch](https://packages.debian.org/search?keywords=python3-dev&searchon=names&suite=stable&section=all)). So this could be updated to Python >=3.5.

Reply:
Maybe the Python 3 line could be moved into the table below?

Reply:
I think the reason it is here and not in the table is because it is a language, and not a library/tool.  But if we think moving it into the below table, or a separate table for the languages make sense, I could change to that instead.

Reply:
😭